src/test/java/GameFieldTest.java

```diff
@@ -8,40 +8,36 @@
 public class GameFieldTest {
 
     @Test
-    public void givenGameObjectOutOfBoundary_whenAddObject_thenCorrect(){
+    public void should_notAddObject_when_objectOutOfBoundary(){
         GameField gameField = new GameField(new Point(10, 10));
 
         gameField.addObject(new GameObject(new Point(9, 9), new Point(2, 2)));
 
         Collection<GameObject> gameObjects = gameField.getGameObjects();
-
-        assertTrue(gameObjects.size() == 0);
+        assertEquals(gameObjects.size(), 0);
```

Comment:
expected value should be on left, actual on right

assertEquals(0, gameObjects.size());